lib/ret_web/channels/hub_channel.ex

```diff
@@ -0,0 +1,86 @@
+defmodule RetWeb.HubChannel do
+  @moduledoc "Ret Web Channel for Hubs"
+
+  use RetWeb, :channel
+
+  alias Ret.{Hub, Repo, SessionStat}
+
+  def join("hub:" <> hub_sid, _payload, socket) do
+    hub = Repo.get_by(Hub, hub_sid: hub_sid)
+    socket |> join_with_hub(hub)
```

Comment:
might be better as:
`Repo.get_by(Hub, hub_sid: hub_sid) |> join_with_hub(socket)`
(and reorder the args in join_with_hub)